app/models/admin_user.rb

```diff
@@ -0,0 +1,6 @@
+class AdminUser < ApplicationRecord
```

Comment:
Are you thinking that AdminUser would have devise and we might have a `User` table that does not? There's a few ways we can do this, and there's pros/cons no matter which direction. Just want to know what you're thinking.

Reply:
ActiveAdmin uses Devise as the default and creates a separate AdminUser model. Then all the reset password, etc. functionality is built in.  And then, yes, I was thinking we would have a separate User model (not connected to Devise), and we could figure out what kind of API authentication is best going forward.

Reply:
ah. got it! 👍 

carry on.